notebooks/Gemma+Gemini+Eval.ipynb

```diff
@@ -0,0 +1,8838 @@
+{
```

Comment:
<p>Can just specify <code>load_in_8bit=True</code> to <code>AutoModelForCausalLM</code> no? Or does it throw deprecation warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/llmops-pipeline/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
It throws depreciation warning and it recommended to use quantization_config in the future instead